scripts/common.sh

```diff
@@ -202,7 +203,49 @@ function diff_values
     shift
     shift
 
-    diff --unified=0 <(echo "$value1") <(echo "$value2") "$@"
+    if ! diff --unified=0 <(echo "$value1") <(echo "$value2") "$@"
+    then
+        if [ "${DIFFVIEW:-}" == "" ]
```

Comment:
Following the logic of my previous comment, this could be:
```suggestion
        if [[ -z "$DIFFVIEW" ]]
```

Reply:
This is also done like this because of potential `set -u`

Reply:
But please do use the other part of the suggestion, i.e. `[[ ]]`.

Same in `diff_files` below.